dbt_semantic_interfaces/protocols/semantic_model.py

```diff
@@ -73,6 +74,17 @@ def description(self) -> Optional[str]:  # noqa: D
     def node_relation(self) -> NodeRelation:  # noqa: D
         pass
 
+    @property
+    @abstractmethod
+    def primary_entity(self) -> Optional[str]:  # noqa: D
```

Comment:
nit: given the docstring, I don't think we need the `# noqa: D` here

Reply:
Removed.